addons/l10n_it_stock_ddt/tests/test_ddt.py

```diff
@@ -32,9 +32,9 @@ def setUpClass(cls, chart_template_ref='l10n_it.l10n_it_chart_template_generic')
         })
 
         settings = cls.env['res.config.settings'].create({})
-        if hasattr(settings, 'button_create_proxy_user'):
-            # Needed when `l10n_it_edi_sdiscoop` is installed
-            settings.button_create_proxy_user()
+        if hasattr(settings, 'button_create_l10n_it_edi_proxy_user'):
```

Comment:
I don't think this hasattr is necessary anymore as it probably depends on l10n_it_edi

Reply:
Ok, trying to remove it.